download_script.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import argparse
```

Comment:
Order suggested by `pylint` among other things:

```suggestion
import argparse
import requests
```